tests/driver_sht2x/Makefile

```diff
@@ -0,0 +1,18 @@
+APPLICATION = driver_sht2x
+include ../Makefile.tests_common
+
+FEATURES_REQUIRED = periph_i2c
+
+USEMODULE += sht2x
+USEMODULE += xtimer
+USEMODULE += printf_float
+
+# set default device parameters in case they are undefined
+TEST_I2C                  ?= I2C_0
+TEST_SHT2X_ADDRESS        ?= 0x40
```

Comment:
Not used. Just get rid of these. Furthermore, the defines from the params file should be over-written

Reply:
This is a remainder of how it was done when I started the sht2x work. I'll remove it